_layouts/base.html.slim

```diff
@@ -93,6 +93,12 @@ body.antialiased
       - unless page.url.start_with? '/news/'
         .large-4.columns.secondary
           - if page.url =~ /^\/docs\/.+/
+            - if page.toc
```

Comment:
"outline" (or, formally, "block_outline") is a pseudo node (meaning it only exists by intent during conversion).